zavod/zavod/exporters/fragment.py

```diff
@@ -49,7 +54,5 @@ def get_inverted(self, id: str) -> Generator[Tuple[Property, Entity], None, None
                 yield prop, entity
 
     def entities(self) -> Generator[Entity, None, None]:
-        for entity in self.view.entities():
-            if entity.id is not None:
-                self._entities[entity.id] = entity
-            yield entity
+        # Don't cache entities here
+        yield from self.view.entities()
```

Comment:
is it maybe even worth making this an exception?

Reply:
Fair, yes it should be